src/mutations/member.ts

```diff
@@ -96,4 +96,49 @@ export default (queryClient: QueryClient, queryConfig: QueryClientConfig) => {
       });
     },
   });
+
+  // mutation to update favorite items of given member
+  queryClient.setMutationDefaults(MUTATION_KEYS.UPDATE_FAVORITE_ITEMS, {
+    mutationFn: (payload) => {
+      const {memberId, itemId, extra, action} = payload;
+      if (action === 'add') {
+        extra.favoriteItems = extra.favoriteItems? extra.favoriteItems.concat([itemId]) : [itemId]
+      }
+      if (action === 'remove') {
```

Comment:
As mentioned in the other PR, let's go with two mutations instead of this if clause.